Sources_v3/StreamChat/Controllers/SearchControllers/UserSearchController.swift

```diff
@@ -145,17 +145,23 @@ public class _ChatUserSearchController<ExtraData: ExtraDataTypes>: DataControlle
     /// - Note: Currently, no local data will be searched, only remote data will be queried.
     ///
     /// - Parameters:
-    ///   - term: Search term.
+    ///   - term: Search term. If empty string or `nil`, all users are fetched.
     ///   - completion: Called when the controller has finished fetching remote data.
     ///   If the data fetching fails, the error variable contains more details about the problem.
-    public func search(term: String, completion: ((_ error: Error?) -> Void)? = nil) {
-        var query = UserListQuery<ExtraData.User>(filter: .or([
-            .autocomplete(.name, text: term),
-            .autocomplete(.id, text: term)
-        ]))
+    public func search(term: String?, completion: ((_ error: Error?) -> Void)? = nil) {
+        var query: UserListQuery<ExtraData.User>
+        if !(term?.isEmpty ?? true) {
+            query = .init(filter: .or([
+                .autocomplete(.name, text: term!),
+                .autocomplete(.id, text: term!)
+            ]))
+        } else {
+            query = .init(filter: .exists(.id)) // Pseudo-filter to fetch all users
+        }
+        // Backend suggest not sorting by name
+        // so we only sort client-side
```

Comment:
This breaks pagination. We can't sort locally differently than on the backend.

Reply:
so I should send the same sorting to the backend?...